cves/kernel/CVE-2020-25211.yml

```diff
@@ -55,7 +55,12 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: |
+  In the Linux kernel through 5.8.7, 
+  a local attacker can inject conntrack netlink configurations that can overflow a 
```

Comment:
Remove project specific things and make this description simpler. The target audience are students who have not taken the class before.